docs/index.md

```diff
@@ -50,8 +53,7 @@ Module           | Information
 
 ## Legal information
 
-* This suite of modules is licensed under the General Public License, version 2
-  or later (GPL-2.0+).
+* This suite of modules is licensed under the General Public License, v2.0 or later (GPL-2.0-or-later).
```

Comment:
Over 80 chars per line (Drupal coding standards also apply to Markdown)

Reply:
I will break down the line